src/components/progress/progress.html

```diff
@@ -0,0 +1,4 @@
+<div class="progress" [style.width.%]="value"></div>
+<div class="animtable"></div>
```

Comment:
I think I made the same typos multiple times ops, its supposed to be `animatable` rather than `animtable`